src/main/java/com/google/devtools/build/lib/bazel/repository/RepositoryOptions.java

```diff
@@ -133,6 +133,19 @@
       help = "If non-empty read the specified resolved file instead of the WORKSPACE file")
   public String experimentalResolvedFileInsteadOfWorkspace;
 
+  @Option(
+      name = "incompatible_use_native_patch",
+      defaultValue = "false",
+      documentationCategory = OptionDocumentationCategory.UNCATEGORIZED,
+      effectTags = {OptionEffectTag.LOADING_AND_ANALYSIS},
+      metadataTags = {
+          OptionMetadataTag.INCOMPATIBLE_CHANGE,
+          OptionMetadataTag.TRIGGERED_BY_ALL_INCOMPATIBLE_CHANGES
+      },
+      help = "If enabled, Bazel's repository rules use a native implementation of patch on Windows, "
```

Comment:
I thought it should say "uses native implementation of patch" - because it is on all platforms?
Also, each time I read "native", I think "native=operating system native". Could we say "Bazel-native" ?